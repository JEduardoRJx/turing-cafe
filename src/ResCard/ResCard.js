import React from 'react'
import './ResCard.scss'

const ResCard = ({ guest }) => {
  return (
    <article id={guest.id}>
      <h2>{guest.name}</h2>
      <p>{guest.date}</p>
      <p>{guest.time}</p>
      <p>Num. of guests: {guest.number}</p>
      <button className='cancel-btn' 
        type='button'>CANCEL</button>
    </article>
  )
}

export default ResCard;