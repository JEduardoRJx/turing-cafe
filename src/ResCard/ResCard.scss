article {
  // background-color: blue;
  height: 300px;
  width: 250px;
}