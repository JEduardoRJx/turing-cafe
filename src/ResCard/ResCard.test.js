import React from 'react';
import ResCard from './ResCard';
import { shallow } from 'enzyme'

describe('ResCard', () => {
  let wrapper, mockGuest;

  beforeEach(() => {
    mockGuest = {
      id: 0,
      name: 'Eduardo',
      date: '05/01',
      number: 2,
      time: '5'
    }
    wrapper = shallow(<ResCard guest={mockGuest}/>)
  });

  it('should create a snapshot of ResCard', () => {
    expect(wrapper).toMatchSnapshot();
  });

})