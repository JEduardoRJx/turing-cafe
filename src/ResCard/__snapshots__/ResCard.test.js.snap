// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResCard should create a snapshot of ResCard 1`] = `
<article
  id={0}
>
  <h2>
    Eduardo
  </h2>
  <p>
    05/01
  </p>
  <p>
    5
  </p>
  <p>
    Num. of guests: 
    2
  </p>
  <button
    className="cancel-btn"
    type="button"
  >
    CANCEL
  </button>
</article>
`;
